src/cls/IPM/ResourceProcessor/Default/Document.cls

```diff
@@ -172,6 +172,26 @@ Method OnPhase(pPhase As %String, ByRef pParams, Output pResourceHandled As %Boo
                 Set tSC = $$$OK
                 Quit
               }
+              If ..ResourceReference.Deploy {
+                If tVerbose {
+                  Write !, "Skipping deployed resource: " _ ..ResourceReference.Name
+                }
+                Set ..CompileFromProject = 1
+                Set tDeployedDirectory = ##class(%File).NormalizeDirectory("studio-project", ..ResourceReference.Module.Root)
```

Comment:
Could all "magic strings" be macros or helper methods? So that includes "studio-project", "__deployed" and "Deployed.xml". Perhaps helper methods to construct the file name for a given module's deployed studio project file + a helper method for the studio project name for a module's deployed code?